markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+
+### What are the resource maximums
+
+All computing HPCs will have some resource limits, depending on the sizes and configuration of the nodes it offers. You should find out what is _largest_ node available to all users of a HPC and record the following information:
+
+- maximum amount of memory (RAM) available
+- maximum number of CPUs (cores) available
+
+<details markdown='1'><summary> SLURM tips</summary>
+
+For example, if you're using the SLURM scheduler, you can run the following command to find specifications of all nodes on your cluster
+
+```bash
+scontrol show node
+```
+
+and look for the node with largest `CPUTot` and `RealMemory` fields and node the values there.
+
+</details>
+</br>
+Furthermore, you should check if there is any overall walltime (time limit) for jobs i.e. is there a maximum time a job can run for before it gets killed by the scheduler.
+</br>
+</br>
+
+<details markdown='1'><summary> SLURM tips</summary>
+
+For example, if you're using the SLURM scheduler, you can run the following command to the walltime of any queue/partitions on your cluster with
+
+```bash
+sinfo
+```
+
+and look for the TIMELIMIT column for the longest running queue that _all_ users have access to.
+</details>
+</br>
+
+### Do you have to specify a queue or partition
+
+Some clusters require your to specify which queue or partition to submit your job to (based on the resource requirements of the job). You should check if these exist, what they are called, and what limits they may have (e.g. walltimes).
+
+You should already consider in which 'order' you would like to use these queues, to make use of the Nextflow and nf-core automated 'retry' system (i.e. if a process runs too long or uses too much memory, it can be automatically resubmitted with more time/memory)
+
+<details markdown='1'><summary> SLURM tips</summary>
+
+For example, if you're using the SLURM scheduler, you can run the following command to the walltime of any queues/partitions on your cluster with
+
+```bash
+sinfo
+```
+
+You should record the names of each `PARTITION` and the corresponding `TIMELIMIT` column. Note there may be additional resource limitations for each queue, so check your institutional cluster's documentation for any other limits.
+
+</details>
+
+## Are there any common resource or data locations on the cluster
+
+For institutions that work on common topics (e.g. genomics), you might have a centralised local copy of the Illumina `iGenome` set of reference genomes. If it exists, you can note down the paths to this directory.
+
+Furthermore, some institutions will specify a common place to store all container images (e.g. for Docker, Singularity etc) in a cache directory. This helps preventing multiple users having their own copies of the same pipeline container image. You should find out if, and where this location exists and note down the path.
```

Comment:
I don't think there is a cache dir option for Docker

Reply:
Huh, just checked the nextflow docs and you're right. That's wierd.